### PR TITLE
Jcs/txn report

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

**Comment by jSylvestre** [comment]
Issue #410

File: Sloth.Web/Sloth.Web.csproj [hunk 1]
```diff
@@ -22,6 +22,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.16" />
```
Comment (sprucely) [inline Sloth.Web/Sloth.Web.csproj:26]: Was there some problem that was solved by adding this?

**Review by sprucely — APPROVED** [review]
Didn't test, but code looks good.

File: Sloth.Web/Sloth.Web.csproj (hunk 1 above)
Comment (jSylvestre) [reply Sloth.Web/Sloth.Web.csproj:26]: No, VS added it when I tried to use the razor page generator

File: Sloth.Core/Domain/Transfer.cs [hunk 2]
```diff
@@ -163,6 +163,8 @@ public string FullObjectToString()
             }
             return result;
         }
+
+        public string ShortFinancialSegmentString => $"{FinancialSegmentString?[0..13]}...";
```
Comment (srkirkland) [inline Sloth.Core/Domain/Transfer.cs:167]: I've never seen this w/o using `string.substr()` but if this works it's ok to me.  clear enough what's happening.

**Review by srkirkland — APPROVED** [review]

File: Sloth.Core/Domain/Transfer.cs (hunk 2 above)
Comment (jSylvestre) [reply Sloth.Core/Domain/Transfer.cs:167]: it takes the first 13 characters of the string. I'll add a comment

**Pull request merged**